LondonBritishLibrary/orient/BLorient539.xml

```diff
@@ -0,0 +1,324 @@
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">ማርያም፡ ስብሕት፡ ቀስተ፡ መሐላሁ፡ ለኖኅ። ዲበ፡ ገጸ፡ ምድር፡ ዳግመ፡ እምደመና፡ ላህ። ኢያንጠብጥብ፡ ዝናመ፡ አይኅ። ሰላም፡ ለኪ።</incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i7">
+                            <locus from="218r"/>
+                            <title type="complete" ref="LIT3990Prayer"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">እሴብሐከ፡ ኦእግዚእየ፡ ወአምላኪየ፡ ኢየሱሰ፡ ክርስቶስ፡ በስብሐተ፡ ማር፡ ፈለስኪኖሰ፡ ጳጳስ፡ ዘሀገረ፡ ምንቡዠ። 
+                                <seg type="supplication">ጸሎቱ፡ ወበረከቱ፡ ወሀብተ፡ ረድኤቱ፡ የሀሉ፡ 
+                                    ምስለ፡ ፍቁሩ፡ <persName ref="PRS13080Isayeyyas">ኢሳይያስ፡</persName> ወፍቅርቱ፡ <persName ref="PRS13081WalattaH">ወለተ፡ ሃይማኖት፡</persName>
+                                    <persName ref="PRS13079HellaweK"><add place="overstrike">ህላዌ፡</add> ክርስቶስ፡</persName> እስከ፡ ለዓለመ፡ ዓለም፡ አሜን።</seg> ስብሐት፡ ለከ፡ ኦእግዚአብሔር፡ አብ፡
+                            </incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i8">
+                            <locus from="223r"/>
+                            <title type="complete" ref="LIT6104MadhanitaN"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><seg type="inscriptio">ጸሎት፡ ወስእለት፡ በእንተ፡ መድኃኒተ፡ ነፍስ፡ ወሥጋ፡</seg> 
+                                ኦእግዚእየ፡ ወአምላኪየ፡ ወመድኃኒየ፡ ኢየሱስ፡ ክርስቶስ።
+                                ወይትከዐው፡ ሞገደ፡ ምሕረትከ፡ ላዕሌየ፤ ከመ፡ ያስጥሞ፡ ለመጽሐፈ፡ ዕዳየ።</incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i9">
+                            <locus from="224r"/>
+                            <title type="complete" ref="LIT6105EgzioA"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">እግዚኦ፡ አድኅነኒ፡ ዘአድኃንኮ፡ ለኤልያስ፡ እምእደ፡ ኤልዛቤል፡ ንግሥት።</incipit>
+                        </msItem>
+                        
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">227</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>10.25</height>
+                                        <width>9</width>
+                                    </dimensions>
+                                </extent>
+                                <condition key="deficient">Mutilated. Several leaves have been torn out between 
```

Comment:
What is mutilated?

Reply:
I copied the catalogue. I think it means material loss.